R/pkg/DESCRIPTION

```diff
@@ -11,7 +11,7 @@ Authors@R: c(person("Shivaram", "Venkataraman", role = c("aut", "cre"),
                     email = "felixcheung@apache.org"),
              person(family = "The Apache Software Foundation", role = c("aut", "cph")))
 URL: http://www.apache.org/ http://spark.apache.org/
-BugReports: https://issues.apache.org/jira/secure/CreateIssueDetails!init.jspa?pid=12315420&components=12325400&issuetype=4
+BugReports: http://issues.apache.org/jira/browse/SPARK
```

Comment:
ah, I thought I tested it logged out. You are right, let's scratch that then.

I like the idea with the wiki - though should that mention when to check with user@spark, when to email dev@spark and when to open a JIRA?